src/atoms/BylineWithTwoLines.js

```diff
@@ -1,69 +1,78 @@
-import Styled from 'styled-components';
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'react-emotion';
 
 import { getColor, getVariable } from '../utils';
 
 import { SvgIcon } from './SvgIcon';
 
-const BylineBlock = Styled.div`
-`;
-const BylineImage = Styled.img.attrs({
-	src: ({ src }) => src,
-})`
-    float: left;
-    width: 4.5rem;
-    height: 4.5rem;
-    overflow: hidden;
-    border-radius: 50%;
-    object-fit: cover;
-    margin-right: ${getVariable('horizontalBase')};
+const BylineBlock = styled.div``;
+
+
+const BylineImage = (props) => {
```

Comment:
Could be a better way, but this way worked :) 

It was implemented with `recompose/withComponent`  but that implementation did not work